appium-tests/common/common.spec.js

```diff
@@ -242,7 +248,6 @@ describe('Contacts Android', function () {
 
     describe('Picking contacts', function () {
         afterEach(function (done) {
-            checkSession(done);
```

Comment:
As far as I remember, this check is to quickly fail without extra requests to the driver. Because when the session is failed to start it could just get unresponsive and we would just sit there and wait for every test to time out.